### ISSUE TITLE
javametrics-prometheus-x.x.x.war does not exist in javametrics-1.0.1.zip

### ISSUE THREAD

**Issue by fmhwong** [RuntimeTools/javametrics#64]
Installation instruction says 

```
Download the latest Application Metrics for Java release from https://github.com/RuntimeTools/javametrics/releases/download/1.0.1/javametrics-1.0.1.zip and this contains:

    javametrics-dash-x.x.x.war - Javametrics Web Application
    javametrics-prometheus-x.x.x.war - Javametrics Prometheus Endpoint
    javametrics-agent-x.x.x.jar - Javametrics agent
```
However, javametrics-prometheus-x.x.x.war doesn't exist in the javametrics-1.0.1.zip

```
felixs-mbp:javaMetrics fmhwong$ ls -R javametrics-1.0.1
agent	client

javametrics-1.0.1/agent:
asm				javametrics-agent-1.0.1.jar

javametrics-1.0.1/agent/asm:
asm-5.0.4.jar		asm-commons-5.0.4.jar	asm-tree-5.0.4.jar

javametrics-1.0.1/client:
webapp

javametrics-1.0.1/client/webapp:
javametrics-dash

javametrics-1.0.1/client/webapp/javametrics-dash:
javametrics-dash-1.0.1.war
```

**Comment by robbinspg** [comment]
This is an issue because the master branch (and instructions) has moved on from the 1.0.1 release.
IMO we need to cut a branch at the level of the release so that the README etc associated with that release level are easily available (other than in the release source zip)

**Comment by robbinspg** [comment]
Fixed in latest 1.1.0 release
